examples/herding_paired_kernel.py

```diff
@@ -0,0 +1,225 @@
+# © Crown Copyright GCHQ
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+Example coreset generation using randomly generated point clouds.
+
+This example showcases how a coreset can be generated from a dataset containing ``n``
+points sampled from ``k`` clusters in space.
+
+A coreset is generated using kernel herding, with a Squared Exponential kernel. This
```

Comment:
Update docstring to state we are using a kernel that combines Squared exponential, PCIMQ and Laplace